source/material_model/reaction_model/grain_size_evolution.cc

```diff
@@ -417,6 +416,18 @@ namespace aspect
         prm.declare_entry ("Advect logarithm of grain size", "false",
                            Patterns::Bool (),
                            "This option does not exist any more.");
+        prm.declare_entry ("Initial step size", "1e-3",
```

Comment:
Can you change the name to make it more clear what this "step size" refers to? Something like
```suggestion
        prm.declare_entry ("Initial step size for ODE solver", "1e-3",
```